jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/PathResource.java

```diff
@@ -373,6 +368,25 @@ private static URI toUri(Path path)
         return pathUri;
     }
 
+    @Override
+    public boolean equals(Object obj)
+    {
+        if (this == obj)
+            return true;
+        if (obj == null)
+            return false;
+        if (getClass() != obj.getClass())
+            return false;
+        PathResource other = (PathResource)obj;
+        return Objects.equals(path, other.path) && Objects.equals(uri, other.uri);
```

Comment:
Isn't it enough just to check uri?  When would the URI be the same but that path different?

Reply:
I've had this thought too.
I was even wondering why we are holding onto the original Path.
Couldn't the PathResource work with only have the "real-path" and "as-requested-uri"?

Reply:
punt to another PR